docs/overrides.md

```diff
@@ -130,6 +130,7 @@ The `value` is the value to set at the `path`. Values can be simple values such
               value:
                 customAnnotation: "customValue"
 ```
+If [importing](../README.md#importingexporting-variables) a variable from another package, that variable can also be used to set a value, using the template syntax `${...}`
```

Comment:
Can we add an explicit example of what this looks like?

Reply:
added in latest commit.